public/scripts/editor/js/bramble-editor.js

```diff
@@ -35,12 +36,18 @@ define(function(require) {
       });
 
       // Start loading the project files
-      Project.load(makeDetails, host, authenticated, function(err, fileToOpen) {
+      Project.load(projectDetails, host, function(err, fileToOpen) {
         if(err) {
           console.error("[Thimble Error]", err);
           return;
         }
 
+        // Initialize the project name UI
+        projectNameComponent = new ProjectRenameUtility(host, authenticated, csrfToken);
```

Comment:
Hmmm... we're also blocking the SSO stuff from being functional while we wait on the editor. I'll do a refactor to have this make more sense.